File: tests/fixtures/structural-index/swift/src/Badge.swift
let BADGE_LIMIT = 12

func renderBadge(_ label: String) -> String {
    return "[\(label)]"
}

class BadgeCard {
    func summarize(_ label: String) -> String {
        return renderBadge(label)
    }

    func renderBadge() -> String {
        return "card"
    }
}

struct BadgeTheme {
    static let accent = "indigo"
}

func orphanAnchor() -> Int {
    return BADGE_LIMIT
}

File: tests/fixtures/structural-index/kotlin/src/Badge.kt
package fixture

const val BADGE_LIMIT = 12

fun renderBadge(label: String): String {
    return "[$label]"
}

interface BadgeRenderer {
    fun render(label: String): String
}

class BadgeCard(private val renderer: BadgeRenderer) {
    private val prefix: String = "badge"

    fun summarize(label: String): String {
        return renderBadge(label) + renderer.render(label)
    }

    fun renderBadge(): String {
        return "card"
    }
}

object BadgeTheme {
    val accent: String = "indigo"
}

fun orphanAnchor(): Int = BADGE_LIMIT
